Profile Folder/chrome/locale/ro/dtd/gsettings.dtd

```diff
@@ -90,10 +90,10 @@
 <!ENTITY gsettings.enableWebsitePinningInEveryEra				"Activați fixarea site-ului în fiecare temă de Chrome in Geckium">
 <!ENTITY gsettings.enableWebsitePinningInEveryEraDescription	"Afișează butonul pentru fixare în fiecare design">
 
-<!ENTITY gsettings.about        "About">
-<!ENTITY gsettings.chooseAbout  "Choose About">
-<!ENTITY gsettings.browser      "Browser">
-<!ENTITY gsettings.ask          "Ask">
+<!ENTITY gsettings.about        "Despre">
+<!ENTITY gsettings.chooseAbout  "Selectează afișarea secțiunei ”despre„">

```

Comment:
Are you having trouble using quotes or is that intentional?

Reply:
those are the quotes used in Romania

Reply:
tho i think ive made a mistake, thank you for pointing it out now lol